maro/simulator/scenarios/vm_scheduling/business_engine.py

```diff
@@ -114,7 +118,13 @@ def _load_configs(self):
 
         self._delay_duration: int = self._config.DELAY_DURATION
         self._buffer_time_budget: int = self._config.BUFFER_TIME_BUDGET
-        self._pm_amount: int = self._config.PM.AMOUNT
+        # Oversubscription rate.
+        self._max_cpu_oversubscription_rate: float = self._config.MAX_CPU_OVERSUBSCRIPTION_RATE / 100
+        self._max_memory_oversubscription_rate: float = self._config.MAX_MEM_OVERSUBSCRIPTION_RATE / 100
+        self._max_utilization_rate: float = self._config.MAX_UTILIZATION_RATE / 100
+        # Load PM related configs.
+        self._pm_amount: int = self._config.PM_AMOUNT
+        self._kill_all_vms_if_overload = self._config.KILL_ALL_VMS_IF_OVERLOAD
```

Comment:
true/false can be used in yaml config

or define the overload handling option as an enum